pokedexDataBaseProject.php

```diff
@@ -21,7 +21,7 @@
         Pokedex
```

Comment:
Looks good, As we discussed though you only accepted story one so there shouldn't be any add functionality in here yet.